src/components/common/ColonyFunding/ColonyFunding.tsx

```diff
@@ -1,9 +1,28 @@
 import React from 'react';
+import { useSetPageBreadcrumbs, useSetPageHeadingTitle } from '~context';
 import FundsTable from '~frame/v5/pages/FundsPage/partials/FundsTable';
+import { formatText } from '~utils/intl';
 
 const displayName = 'common.ColonyFunding';
 
-const ColonyFunding = () => <FundsTable />;
+const ColonyFunding = () => {
+  useSetPageHeadingTitle(formatText({ id: 'incomingFundsPage.title' }));
+  useSetPageBreadcrumbs([
```

Comment:
Content should be wrapped in `useMemo` otherwise breadcrumbs will change on every rerender